src/test/java/org/jitsi/meet/test/ActiveSpeakerTest.java

```diff
@@ -91,9 +91,18 @@ private void assertOneDominantSpeaker(WebDriver driver)
             driver.findElements(By.xpath(
                 "//span[contains(@class,'dominant-speaker')]"
             ));
+        List<WebElement> nonStage = new ArrayList<>();
+        for (WebElement el : dominantSpeakerIndicators)
```

Comment:
I'm wondering can we do the whole thing with XPath only?

Maybe we can have `id`s on the filmstrip containers and include in the XPath that there must be a parent with a specific `id` or maybe even use the current class of the stage filmstrip container in order to not include it in the XPath selection?

Reply:
Yes, I think using xpath is a good idea. I'll update the code